docs/governance/community-guidelines/content-standards-and-practices/release-assets.md

```diff
@@ -0,0 +1,41 @@
+# Release Assets
+
+Each asset released by CCC will be a set of common features, threats, and controls for a category of cloud services.
```

Comment:
This seems to read as the asset being released includes only common features, threats and controls - where "common" here means "not specific to the service category" - the release would also include features, threats and controls that are specific to the service category, right?

Reply:
Good catch, the word "common" is getting overloaded.